slave.go

```diff
@@ -401,6 +567,44 @@ func (s *Slave) nextEOF() (*Packet, error) {
 	return nil, io.EOF
 }
 
+// for Tarantool >= 1.7.7 heartbeat sends encoded vclock to master every second
+func (s *Slave) heartbeat() {
+	if s.version() < versionID(1, 7, 7) {
+		return
+	}
+
+	var (
+		err error
+		pp  *packedPacket
+	)
+
+loop:
+	for {
+		select {
+		case <-time.Tick(time.Second):
+			if pp, err = s.newPacket(&VClock{
+				VClock: s.VClock[1:],
+			}); err != nil {
+				break loop
+			}
+
+			if err = s.send(pp); err != nil {
+				pp.Release()
+				break loop
+			}
+
+			pp.Release()
+
+		case <-s.stop:
```

Comment:
Do we really need extra stop mechanism (channel, sync.Once) in slave?
Who is responsible for closing this channel?